extensions/liquibase-mongodb/runtime/src/main/java/io/quarkus/liquibase/runtime/LiquibaseMongodbConfig.java

```diff
@@ -0,0 +1,68 @@
+package io.quarkus.liquibase.runtime;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+/**
+ * The liquibase configuration
+ */
+@ConfigRoot(name = "liquibase-mongodb", phase = ConfigPhase.RUN_TIME)
+public class LiquibaseMongodbConfig {
```

Comment:
All the properties in this config root class are missing the @ConfigItem annotation. Is that intentional? I think we have to add the annotation.

/cc @loicmathieu @geoand

Reply:
Thanks, this is embarassing 😕 
I must say: TooManyCopyPaseException ...